index.js

```diff
@@ -2535,6 +2545,16 @@
       [$.FiniteNumber, $.FiniteNumber, $.FiniteNumber],
       function(a, b) { return a + b; });
 
+  //#  sum :: Foldable f => f FiniteNumber -> FiniteNumber
+  //.
+  //. Returns the sum of a list of (finite) numbers.
```

Comment:
In light of #180, let's `s/a list/an array/`.

Also, `sum = K(0)` would fit with this description, as it always returns the sum of `[]`, an array of finite numbers. ;)

Let's write *the array* rather than *an array*. I suggest:

_Returns the sum of the given array of (finite) numbers._